hvplot/converter.py

```diff
@@ -1330,7 +1330,10 @@ def method_wrapper(ds, x, y):
                 opts['rescale_discrete_levels'] = self._plot_opts['rescale_discrete_levels']
         else:
             operation = rasterize
-            eltype = 'ImageStack' if self.by else 'Image'
+            if hv.__version__ < '1.18.0':
```

Comment:
This does not work always work. `"1.18.0" < "1.2.2"`.

```suggestion
            if Version(hv.__version__) < Version('1.18.0a1'):
```

where Version is `from packaging.version import Version`. `a1` is because this should work with the a1 release.